Gui/MessageBox.cpp

```diff
@@ -112,11 +114,13 @@ MessageBox::init(const QString & title,
         //QFont f(appFont,appFontSize);
         //_imp->label->setFont(f);
     } else {
+        setSizeGripEnabled(true);
         _imp->infoEdit = new QTextEdit;
         _imp->infoEdit->setFocusPolicy(Qt::NoFocus);
         _imp->infoEdit->setReadOnly(true);
         _imp->infoEdit->setAlignment(Qt::AlignVCenter | Qt::AlignLeft);
-        _imp->infoEdit->append(message);
+        _imp->infoEdit->setWordWrapMode(QTextOption::WordWrap);
+        _imp->infoEdit->setHtml(message);
```

Comment:
It looks the label path above will auto detect rich text or not. I wonder if this code should be something like the following to mimic that behavior.
if (Qt::mightBeRichText(message)) {
  _imp->infoEdit->setHtml(message);
} else {
 _imp->infoEdit->setPlainText(message);
}

Reply:
Sorry for the late reply, I will add check for rich text sometime tomorrow after work.

Reply:
Any update @rodlie ?

Reply:
ohh, forgot about this one, sorry. Will add fix as soon as I'm in front of my dev machine (~1h).